src/backend/utils.ts

```diff
@@ -86,7 +85,6 @@ import {
   updateWineVersionInfos,
   wineDownloaderInfoStore
 } from './wine/manager/utils'
-import { clean } from 'easydl/dist/utils'
```

Comment:
Is this change related? also the translations update? or it came from another branch?

Reply:
These variables were unused. Not sure how they got back into develop after your replaced unzipper with the other library.